Author: =?UTF-8?q?Jean-Fran=C3=A7ois=20Fortin=20Tam?= <nekohayo@gmail.com>
Last-Update: 2013-03-16
Forwarded: yes
Origin: upstream, http://git.dyne.org/frei0r/commit/?id=245bb38
Description: Make property names for "pixeliz0r" and "lenscorrection"
 human-readable

---
 src/filter/lenscorrection/lenscorrection.c | 10 +++++-----
 src/filter/pixeliz0r/pixeliz0r.c           |  4 ++--
 2 files changed, 7 insertions(+), 7 deletions(-)

diff --git a/src/filter/lenscorrection/lenscorrection.c b/src/filter/lenscorrection/lenscorrection.c
index 3db5541..85a4a54 100644
--- a/src/filter/lenscorrection/lenscorrection.c
+++ b/src/filter/lenscorrection/lenscorrection.c
@@ -62,27 +62,27 @@ void f0r_get_param_info(f0r_param_info_t* info, int param_index)
 	switch(param_index)
 	{
 		case 0:
-			info->name = "xcenter";
+			info->name = "X center";
 			info->type = F0R_PARAM_DOUBLE;
 			info->explanation = "";
 			break;
 		case 1:
-			info->name = "ycenter";
+			info->name = "Y center";
 			info->type = F0R_PARAM_DOUBLE;
 			info->explanation = "";
 			break;
 		case 2:
-			info->name = "correctionnearcenter";
+			info->name = "Correction near center";
 			info->type = F0R_PARAM_DOUBLE;
 			info->explanation = "";
 			break;
 		case 3:
-			info->name = "correctionnearedges";
+			info->name = "Correction near edges";
 			info->type = F0R_PARAM_DOUBLE;
 			info->explanation = "";
 			break;
 		case 4:
-			info->name = "brightness";
+			info->name = "Brightness";
 			info->type = F0R_PARAM_DOUBLE;
 			info->explanation = "";
 			break;
diff --git a/src/filter/pixeliz0r/pixeliz0r.c b/src/filter/pixeliz0r/pixeliz0r.c
index 6baf91b..e13805d 100644
--- a/src/filter/pixeliz0r/pixeliz0r.c
+++ b/src/filter/pixeliz0r/pixeliz0r.c
@@ -47,12 +47,12 @@ void f0r_get_param_info(f0r_param_info_t* info, int param_index)
   switch(param_index)
     {
     case 0:
-      info->name = "BlockSizeX";
+      info->name = "Block width";
       info->type = F0R_PARAM_DOUBLE;
       info->explanation = "Horizontal size of one \"pixel\"";
       break;
     case 1:
-      info->name = "BlockSizeY";
+      info->name = "Block height";
       info->type = F0R_PARAM_DOUBLE;
       info->explanation = "Vertical size of one \"pixel\"";
       break;      
-- 
2.0.0.rc2

